packages/hono/test/unit/build.test.ts

```diff
@@ -102,7 +108,7 @@ const fixtures = {
   },
   '10-index-ts-no-tsconfig': {
     handler: ['index.js'],
-    moduleType: 'cjs',
+    moduleType: 'esm',
```

Comment:
Just confirming that `cjs` behavior was a bug for this test case

Reply:
Yes! 👍